data/org.freedesktop.portal.GameMode.xml

```diff
@@ -149,6 +149,55 @@
         <arg type="i" name="result" direction="out"/>
       </method>
 
+      <!-- PidFd based APIs -->
+      <!--
+          QueryStatusByPIDFd:
+          @target: Pidfd to query the GameMode status of
+          @requester: Pidfd of the process requesting the information
+          @result: The GameMode status for @target
+
+          Query the GameMode status for a process.
+	  Like org.freedesktop.portal.GameMode.QueryStatusByPid() but @requester
+          and @target are pidfds representing the processes.
+      -->
+      <method name="QueryStatusByPIDFd">
+        <arg type="h" name="target" direction="in"/>
+	<arg type="h" name="requester" direction="in"/>
```

Comment:
Indentation mishap here. More seriously, the args need an 'fd' annotation.

Reply:
Ah of course, totally forgot about the 'fd' annotation. Thanks.